MAINTAINERS

```diff
@@ -0,0 +1,38 @@
+
+        List of maintainers for mdadm
```

Comment:
If you don't have preference please convert it to markdown. Please add hyperlink from main README to this file.